### PR TITLE
[JENKINS-32670] Integration test

### PR DESCRIPTION
**Pull request by jglick**
Downstream of https://github.com/jenkinsci/branch-api-plugin/pull/32.

@reviewbybees

### CODE REVIEW

**Comment by kzantow** [comment]
:bee: there must be a way to not depend on snapshots, though; or do you (or someone else) just go and update these all at the same time when they're published?

File: multibranch/src/test/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectTest.java [hunk 1]
```diff
@@ -123,6 +127,7 @@
         // RateLimitBranchProperty & BuildRetentionBranchProperty hidden by JobPropertyStep.HideSuperfluousBranchProperties.
         // UntrustedBranchProperty hidden because it applies only to Project.
         assertEquals(Collections.<Class<? extends BranchProperty>>emptySet(), clazzes);
+        assertEquals(Collections.<BranchPropertyStrategyDescriptor>emptyList(), r.jenkins.getDescriptorByType(BranchSource.DescriptorImpl.class).propertyStrategyDescriptors(p, r.jenkins.getDescriptorByType(SingleSCMSource.DescriptorImpl.class)));
```
Comment (amuniz) [inline multibranch/src/test/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectTest.java]: I'm ready to see this line without scrolling.

image

**Comment by amuniz** [comment]
:bee:

@kzantow depending on snapshots is required when working with upstream changes. Once the upstream is released the snapshot is updated and this PR can be merged.

**Comment by jglick** [comment]
> Once the upstream is released the snapshot is updated and this PR can be merged.

Or I just merge with an `@Ignore`.

**Pull request merged**